e2e/tests/test_cases/single_replica_node_down.robot

```diff
@@ -0,0 +1,118 @@
+*** Settings ***
+Documentation    Single replica node down
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-down/single-replica-node-down/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/host.resource
+
+Library    OperatingSystem
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+${SHUTDOWN_WAIT_DURATION}    180
+
+*** Test Cases ***
+Single Replica Node Down Deletion Policy do-nothing With RWO Volume Data Locality disabled
+    When Create persistentvolumeclaim 0 using RWO volume
+    And Create deployment 0 with persistentvolumeclaim 0
+    And Write 100 MB data to file data in deployment 0
+    And Update persistentvolumeclaim 0 volume replica count to ${1}
```

Comment:
How about `Update volume of deployment 0 replica count to`?

Since we already have many keywords like `volume of ${workload_kind} ${workload_id}`.

WDYT? Does it make sense to you?

Reply:
Updated